### PR TITLE
Adds inline CTA for design checklist to blog

### PR DESCRIPTION
**Pull request by mnaumanali94**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 65e15e476f176b0c2decd89728fb031c18beebdc

https://deploy-preview-1677--stoplightio.netlify.app

File: netlify/blog-posts/api-design-patterns-rest.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ One thing to specifically avoid with friendly REST endpoint names is describing
 
 Our [CRUD API Design Recommendations](https://stoplight.io/blog/crud-api-design/) goes into more detail, including popular topics like plurals and versioning.
 
+<a href="/api-design-checklist-download">![The Stoplight Platform](/images/cta-buttons/blog-cta-design-checklist.svg)</a>
```
Comment (paulatulis) [inline netlify/blog-posts/api-design-patterns-rest.md:87]: Any reason to consider adding a UTM to these?

File: netlify/lists/blog-design.yaml [hunk 2]
```diff
@@ -1,8 +1,20 @@
 path: /blog/api-design
 tag: blog-design
-title: API Design Blog Posts
+title: THE API DESIGN BLOG
+subtitle: >-
+  The #1 source of tips and tricks about building quality APIs with Design-First
```
Comment (paulatulis) [inline netlify/lists/blog-design.yaml:5]: What about 
The #1 source of tips and tricks for building quality APIs with a Design-First Approach
or 
The #1 source of tips and tricks for building quality, Design-First APIs

File: netlify/lists/blog-documentation.yaml [hunk 3]
```diff
@@ -1,8 +1,20 @@
 path: /blog/api-documentation
 tag: blog-documentation
-title: Latest API Documentation & API Docs Blog Posts
+title: THE API DESIGN BLOG
+subtitle: >-
+  The #1 source of tips and tricks about building quality APIs with Design-First
```
Comment (paulatulis) [inline netlify/lists/blog-documentation.yaml:5]: Same as above

File: netlify/lists/blog-testing.yaml [hunk 4]
```diff
@@ -1,8 +1,20 @@
 path: /blog/api-testing
 tag: blog-testing
-title: API Testing Blogs
+title: THE API DESIGN BLOG
+subtitle: >-
+  The #1 source of tips and tricks about building quality APIs with Design-First
```
Comment (paulatulis) [inline netlify/lists/blog-testing.yaml:5]: Same as above

File: netlify/lists/blog.yaml [hunk 5]
```diff
@@ -1,8 +1,20 @@
-title: Latest API Blogs From Stoplight
+title: THE API DESIGN BLOG
 subtitle: >-
-  Musings on API design, testing, documentation, trends, and related technology.
+  The #1 source of tips and tricks about building quality APIs with Design-First
```
Comment (paulatulis) [inline netlify/lists/blog.yaml:3]: Same as above

File: netlify/lists/featured.yaml [hunk 6]
```diff
@@ -1,8 +1,20 @@
 path: /blog/featured
 tag: blog-featured
-title: Featured Blog Posts
+title: THE API DESIGN BLOG
+subtitle: >-
+  The #1 source of tips and tricks about building quality APIs with Design-First
```
Comment (paulatulis) [inline netlify/lists/featured.yaml:5]: Same as above

File: netlify/lists/general.yaml [hunk 7]
```diff
@@ -1,8 +1,20 @@
 path: /blog/general
 tag: blog-general
-title: General Blog Posts
+title: THE API DESIGN BLOG
+subtitle: >-
+  The #1 source of tips and tricks about building quality APIs with Design-First
```
Comment (paulatulis) [inline netlify/lists/general.yaml:5]: Same as above

File: netlify/lists/stoplight-webinars.yaml [hunk 8]
```diff
@@ -1,6 +1,7 @@
 path: /video
 tag: video
-title: Stoplight Video Content
+title: Stoplight Video Hub
+subtitle: Tips and tricks on API Design, Upcoming Webinars and Customer Testimonials.
```
Comment (paulatulis) [inline netlify/lists/stoplight-webinars.yaml:4]:
```suggestion
subtitle: Tips and tricks on API Design, Upcoming Webinars, and Customer Testimonials.
```

Comment (paulatulis) [reply netlify/lists/stoplight-webinars.yaml:4]: Also noticed that sometimes these subtitles have a period and other times, they don't. Suuuuuper trivial but will be good to stay consistent in style.

File: src/components/Tabs/index.tsx [hunk 9]
```diff
@@ -18,11 +18,11 @@ export interface ITabs {
 export const Tab: React.FunctionComponent<ITab> = ({ isActive, href, title }) => {
```
Comment (paulatulis) [inline src/components/Tabs/index.tsx:18]: The tabs look like they are floating, but I assume that's intended? 

Screen Shot 2020-12-23 at 12 40 59 PM

**Review by paulatulis — COMMENTED** [review]
This looks incredible!

Question: are you planning to hold off on merging this until the design checklist pdf is finalized?

File: netlify/blog-posts/api-design-patterns-rest.md (hunk 1 above)
Comment (mnaumanali94) [reply netlify/blog-posts/api-design-patterns-rest.md]: Added. Good idea!

File: src/components/Tabs/index.tsx (hunk 9 above)
Comment (mnaumanali94) [reply src/components/Tabs/index.tsx:18]: It is! Looks fine, no?

**Comment by mnaumanali94** [comment]
> This looks incredible!
> 
> Question: are you planning to hold off on merging this until the design checklist pdf is finalized?

I can update the PDF in hubspot. Looking to get this out though.

**Review by paulatulis — APPROVED** [review]

**Pull request merged**